### PR TITLE
Added and Fixed content.

### PR DESCRIPTION
**Pull request by thispsj**
Added Esper, Openode and RedHat for Devs.
Fixed my mistake in Microsoft Teams.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1164,7 +1166,8 @@ Table of Contents
   * [education.github.com](https://education.github.com/pack) — Collection of free services for students. Registration required
   * [Framacloud](https://degooglisons-internet.org/en/list/) — A list of Free/Libre Open Source Software and SaaS by the French non-profit [Framasoft](https://framasoft.org/en/).
   * [getawesomeness](https://getawesomeness.herokuapp.com) — Retrieve all amazing awesomeness from GitHub... a must see
-  * [github.com — FOSS for Dev](https://github.com/tvvocold/FOSS-for-Dev) — A hub of free and Open Source software for developers
+  * [github.com — FOSS for Dev](https://github.com/tvvocold/FOSS-for-Dev) — A hub of free and Open Source software for developers.
```
Comment (ripienaar) [inline README.md:1169]: Please add this to the existing github entry if not already there.

Comment (thispsj) [reply README.md:1169]: @ripienaar I did not add that entry. It already exists. I just added some punctuation to it.

Comment (thispsj) [reply README.md:1169]: If you want I can revert that back.

Comment (ripienaar) [reply README.md:1169]: ah indeed, no its fine I missread, no problem

Comment (thispsj) [reply README.md:1169]: No problem. Happens.

**Pull request merged**